puppet/modules/jenkins_job_builder/manifests/config.pp

```diff
@@ -22,29 +22,13 @@
     notify  => Exec["jenkins-jobs-update-${config_name}"],
   }
 
-  cron { "jenkins-jobs-update-${config_name}-delete-old":
```

Comment:
those should be `ensure => absent` to really remove them.

Reply:
Ahh right, the ole absent then remove cycle. I forget that some(every)times.